src/flamegraph/svg.rs

```diff
@@ -118,16 +125,17 @@ where
     ))?;
     svg.write_event(Event::CData(BytesText::from_escaped_str(&format!(
         "\
-var nametype = 'Function:';
+var nametype = '{}';
 var fontsize = {};
 var fontwidth = {};
 var xpad = {};
 var inverted = {};
 var searchcolor = 'rgb(230,0,230)';",
-        super::FONTSIZE,
-        super::FONTWIDTH,
+        opt.name_type.replace("'", "\\'"),
```

Comment:
```suggestion
        opt.name_type.replace('\'', "\\'"),
```

Reply:
Otherwise I suspect clippy will complain. Specifically, `replace` is faster if the pattern is a `char` than if it's a single-character `str`.

Reply:
clippy does not complain :)
But I found another case escaping case - `\`

Reply:
Done with `enquote` crate.